src/main.py

```diff
@@ -0,0 +1,73 @@
+from typing import Any, Dict, List
+
+import gymnasium as gym
+import highway_env as highway
+from bppy import *
+
+# BPpy Events
+
+def create_env(config: Dict[str, Any]) -> gym.Env:
+    env = gym.make('highway-v0', render_mode='rgb_array', config=config)
+    env.reset()
+    return env
+
+
+def set_config():
+    config = {
+        "centering_position": [0.5, 0.5],
+        "vehicles_count": 1,
+        "controlled_vehicles": 7,
+        "lanes_count": 4,
+        "initial_positions": [
+            [15, 2, 32],  # (x_position, lane_index, speed)
+            [45, 1, 20],
+            [65, 2, 27],
+            [105, 0, 15],
+            [135, 3, 35],
+            [165, 1, 17],
+            [195, 3, 23],
+            [225, 0, 29]
+        ],  # Fixed start positions WIP
+        "observation": {
+            "type": "MultiAgentObservation",
+            "observation_config": {
+                "type": "Kinematics",
```

Comment:
Sorry, dass war von mir verwirrend geschrieben. Ich meinte vorhin nur die feature_range, den "vehicle_count" und die features würden wir weiterhin benötigen.
```suggestion
                "type": "Kinematics",
                "vehicles_count": 1,
                "features": ["x", "y", "vx", "vy"],
```